rotkehlchen/api/rest.py

```diff
@@ -1842,3 +1842,33 @@ def get_asset_icon(
             response.set_etag(hashlib.md5(image_data).hexdigest())
 
         return response
+
+    def _sync_data(self, action: Literal['upload', 'download']) -> Dict[str, Any]:
+        try:
+            success, msg = self.rotkehlchen.premium_sync_manager.sync_data(action)
+            return _wrap_in_result(success, message=msg)
+        except RemoteError as e:
+            return wrap_in_fail_result(str(e), status_code=HTTPStatus.BAD_GATEWAY)
+        except PremiumApiError as e:
+            return wrap_in_fail_result(str(e), status_code=HTTPStatus.BAD_GATEWAY)
+        except PremiumAuthenticationError as e:
+            return wrap_in_fail_result(str(e), status_code=HTTPStatus.UNAUTHORIZED)
+
+    def sync_data(
+            self,
+            async_query: bool,
+            action: Literal['upload', 'download'],
+    ) -> Response:
+        if async_query:
+            return self._query_async(
+                command='_sync_data',
+                action=action,
+            )
+
+        result_dict = self._sync_data(action)
+
+        status_code = result_dict['status_code']
```

Comment:
This is wrong here. It breaks for all sucesful calls. In python this raises an exception. To achieve what you want just change to:
```python
        status_code = result_dict.get('status_code', None)
```